### DISCUSSION TITLE
How to debug external libraries?

### DISCUSSION THREAD

**Discussion by wybryk-ananasa** [mfussenegger/nvim-jdtls#564] [Q&A] [answered]
I would like to enter external libraries' code while debugging (step-into). What do I need to do?
Or how to setup language server so that I can enter external library code?

**Comment by mfussenegger** [comment, marked as answer]
This should just work if you followed the instructions in the readme for your setup

**Reply by wybryk-ananasa** [reply]
Yep it does, thanks.
